unix/w0vncserver/portals/Portal.h

```diff
@@ -0,0 +1,37 @@
+#ifndef __PORTAL_H__
+#define __PORTAL_H__
+
+#include <stdint.h>
+
+#include <gio/gio.h>
+
+class Portal {
```

Comment:
I'm a bit confused about the abstraction here. Is this a class representing a dbus connection? Or a specific portal service? It seems to be halfway between both.

It looks like it would fit better as an abstraction for the portal, as the core interface seems to be `call()`. I.e. let this `Portal` class manage the `GDBusProxy` object.